source/renderer/app/stores/NetworkStatusStore.js

```diff
@@ -130,6 +145,15 @@ export default class NetworkStatusStore extends Store {
     if (!this.isConnected) this._updateNetworkStatus();
   };
 
+  _updateNetworkStatusWhenReconnected = async () => {
+    if (this.isConnected && this.hasBeenConnected) {
```

Comment:
@DominikGuzei we should not trigger this check in case user has already decided to ignore the NTP checks - AKA. if `isSystemTimeIgnored` is `true`...